src/utils/options.ts

```diff
@@ -0,0 +1,46 @@
+import { OptionValidationResult } from "domain/entities/OptionValidationResult";
+import _ from "lodash";
+
+function escapeCsv(value: string): string {
+    return `"${value.replace(/"/g, '""')}"`;
+}
```

Comment:
CSV is a simple format, but still, a complete implementation has to take in account quotes, multi-lines and so on. I'd just use a csv library.